src/saveload/saveload.h

```diff
@@ -377,6 +377,7 @@ enum SaveLoadVersion : uint16_t {
 	SLV_DEPOT_UNBUNCHING,                   ///< 331  PR#11945 Allow unbunching shared order vehicles at a depot.
 	SLV_AI_LOCAL_CONFIG,                    ///< 332  PR#12003 Config of running AI is stored inside Company.
 	SLV_SCRIPT_RANDOMIZER,                  ///< 333  PR#12063 v14.0 Save script randomizers.
+	SLV_VEHICLE_ECONOMY_AGE,                ///< 334  PR#12141 Add vehicle age in economy year, for profit stats minimum age
```

Comment:
Nitpick: missing dot at end of sentence.

Reply:
Since this is going to be backported, should we add the v14.0 tag on this line and change the v14.0 on the previous line to v14.0-RC1?
Or is that something to not do until the backport? With the risk that it gets forgotten.

Reply:
When we do it at the backport, the master is wrong. So I agree, let's do it in this PR :)